src/_includes/banner.html

```diff
@@ -3,6 +3,12 @@
   {% assign fl_url = '/create' -%}
   <div class="site-banner site-banner--default" role="alert">
     {% comment %} For headings use <h3> elements. {% endcomment -%}
-    Looking for inspiration? <a href='https://www.youtube.com/watch?v=MI7Rl2umeeU'>See examples of what you can build with Flutter!</a>
+    Looking for inspiration?<br>
+    Check out the
+    <a href="https://www.youtube.com/watch?v=MI7Rl2umeeU">Flutter
+    Vignettes video</a><br>
+    created by the noted design firm, 
```

Comment:
No commas around "gskinner"... If you remove the following phrase, it changes the meaning of the sentence.

Maybe delete "created" and "See what's possible in Flutter!"?